src/lib/functions/server.ts

```diff
@@ -348,7 +350,11 @@ export const startFunctionsServer = async (
   } else {
     // The order of the function directories matters. Rightmost directories take
     // precedence.
```

Comment:
same here, is it possible to test this?

Reply:
Same answer as above. We end up testing this in the different commands by creating conflicting function and checking the output of the one that wins.